src/extensions/nuget/Microsoft.DotNet.UpgradeAssistant.Extensions.NuGet/PackageLoader.cs

```diff
@@ -455,10 +455,14 @@ public bool CreateArchive(IExtensionInstance extension, Stream stream)
             {
                 Id = extension.Name,
                 Version = NuGetVersion.Parse(extension.Version.ToString()),
-                Description = extension.Description,
-                PackageTypes = new[] { new PackageType("UpgradeAssistantExtension", new Version(1, 0, 0)) }
+                Description = extension.Description
             };
 
+            if (packageType is not null)
+            {
+                builder.PackageTypes = new[] { new PackageType(packageType, new Version(1, 0, 0)) };
+            }
```

Comment:
No, this is v1 of the package type. We don't want that configureable because that's for us to manage what it means. They can define their own version the same way they always did